Tell doxygen not to generate dir_*.html pages for directory entries above
$top_srcdir, so instead builds generate the same dir_*.html pages no matter
what the path to the build workspace is.

Need to submit upstream.

--- dbus-1.10.22/Doxyfile.in	2017-06-29 10:47:05.000000000 -0700
+++ dbus-1.10.22/Doxyfile.in	2017-08-15 17:53:15.519774584 -0700
@@ -16,7 +16,7 @@
 REPEAT_BRIEF           = YES
 ALWAYS_DETAILED_SEC    = NO
 FULL_PATH_NAMES        = NO
-STRIP_FROM_PATH        = 
+STRIP_FROM_PATH        = @top_srcdir@
 INTERNAL_DOCS          = NO
 STRIP_CODE_COMMENTS    = YES
 CASE_SENSE_NAMES       = YES
